src/AppBundle/Controller/Admin/BlogController.php

```diff
@@ -134,19 +135,16 @@ public function showAction(Post $post)
      * @Route("/{id}/edit", requirements={"id": "\d+"}, name="admin_post_edit")
      * @Method({"GET", "POST"})
      */
-    public function editAction(Post $post, Request $request)
+    public function editAction(Request $request, EntityManagerInterface $em, Post $post, Slugger $slugger)
```

Comment:
I don't know. To me Request and the Doctrine thing look better at the beginning. But this is just an opinion.